src/pythermalcomfort/models.py

```diff
@@ -3737,12 +3738,12 @@ def _reset_setpt(self):
         # Set operative temperature under PMV=0 environment
         # par = 1.25
         # 1 met = 58.15 W/m2
-        met = self.bmr * 1.25 / 58.15  # [met]
+        met = self.bmr * par / 58.15  # [met]
         self.to = self._calculate_operative_temp_when_pmv_is_zero(met=met)
         self.rh = 50  # Relative humidity
         self.v = 0.1  # Air velocity
         self.clo = 0  # Clothing insulation
-        self.par = 1.25  # Physical activity ratio
+        self.par = par  # Physical activity ratio
```

Comment:
Shall we rename `par` into `physical_activity_ratio` at first glance I thought that par meant parameter.

Reply:
I think it can be left as is because this is a common abbreviation like PMV.